src/components/App.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from "react";
+import LocationDetails from "./LocationDetails";
+import ForecastSummaries from "./ForecastSummaries";
+import ForecastDetails from "./ForecastDetails";
+import getForecast from "../requests/getForecast";
+import SearchForm from "./SearchForm";
+import "../styles/App.css";
+
+function App() {
+  const [selectedDate, setSelectedDate] = useState(0);
+  const [forecasts, setForecasts] = useState([]);
+  const [location, setLocation] = useState({ city: "", country: "" });
+  const [searchText, setSearchText] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+
+  useEffect(() => {
+    getForecast(setSelectedDate, setForecasts, setLocation);
+  }, []);
+
+  const handleCitySearch = () => {
+    getForecast(setSelectedDate, setForecasts, setLocation, searchText, setErrorMessage);
+  };
+
+  const selectedForecast = forecasts.find((item) => item.date === selectedDate);
+  const { city, country } = location;
+  const handleForecastSelect = (date) => {
+    setSelectedDate(date);
+  };
+  return (
```

Comment:
Generally its a good idea to leave a clear line before the return statement inyour component so its easy to see at a glance where the output is going to be
```
  };

  return (
```